### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by zFranqDEV** [xKaYoz/AkarianAuctionHouse#5]
Hello, why when placing /ah gives me errors in the console, please help me

**Comment by xKaYoz** [comment]
Hey there! Would you mind to send me the errors that are in the console! Thanks!

**Comment by zFranqDEV** [comment]
[https://pastebin.com/d5mAwUHf](url)

**Comment by xKaYoz** [comment]
Hey there zFranqDEV!

Sorry for the late response, the holiday season is always a mess at work. The issue I found is that the NameManager class that I use uses API calls to the Mojang servers to get the most up to date name for a player. If there is a disruption in your connection to their servers it seems to throw this or a similar error.

In the next update that I will be posting, I believed to have fixed this issue by checking if the connection is there is an error in fetching the Mojang API servers' data, I have made it just default to using the built in Bukkit API which should fix our issue.

Once again I do apologize about the late response, but I appreciate the report and if you have any more issues feel free to open another ticket!
